src/controllers/booking.controller.js

```diff
@@ -11,6 +14,28 @@ class bookingController {
         room_id: roomId
       };
       const booking = await bookingService.createBooking(data);
+      const user = await new UserService().getUserId(req.user.id)
```

Comment:
could you try to get user data from req.user instead of getting him from database since you are loggedIn unless you are getting onther user.